README.md

```diff
@@ -42,9 +42,9 @@ Spartacus provides core storefront features such as:
 # Requirements
 
 - SAP Commerce Cloud instance (Release 1808)
-- Angular CLI (https://angular.io/): >= 6.2.4 and < 7.0.0
-- node.js >= 8.9.0
-- yarn >= 1.6.0
+- Angular CLI (https://angular.io/): >= v7.2.1 and < 8.0.0
```

Comment:
It's small bu why does 7.2.1 get a `v` 😉 ?

Reply:
The more consistent the better.

7.2.1 isn't that special.  I removed the v 🙂.